derive-new-test/Cargo.lock

```diff
@@ -1,38 +0,0 @@
-[root]
```

Comment:
I prefer to keep Cargo.lock files checked in. There is no real benefit to not checking them in.